componentMap.js

```diff
@@ -1,5 +1,10 @@
-import ComponentMap from '@irvingjs/styled-components';
+import createThemedComponentMap from '@irvingjs/styled-components/componentMap';
+import { componentMap as WordPressComponentMap } from '@irvingjs/wordpress';
+import { IntegrationsManager } from '@irvingjs/integrations';
```

Comment:
These 2 packages `@irvingjs/integrations` and `@irvingjs/wordpress` aren't in the `package.json` on this repo

Reply:
Oh, good catch! That brings up the question: do we want the wordpress package included here?

Reply:
I'm not sure, @jameswburke @joemcgill @ostowe any preferences?

Reply:
I didn't include it originally, and once we have a CLI it'll probably be a flag. But 99% of people looking at this are likely going to want it, and it's a lot easier to remove if you don't need it, vs. figuring out _what_ needs to be added.

tldr; let's add it imo